examples/progressbar/progressbar/app.py

```diff
@@ -65,3 +65,6 @@ def decrease_progress2(self, button, **kw):
 def main():
     # App name and namespace
     return ProgressBarApp('ProgressBar', 'org.pybee.examples.progressbar')
+
+if __file__ == '__main__':
+    main().main_loop()
```

Comment:
This new block shouldn't be required - and if it were, the usual idiom would be `if __name__ == '__main__':`; what's the purpose behind this?

Reply:
On doing `python -m app` nothing happens for me.
```
venv) yugesh@yugesh-HP-240-G1-Notebook-PC:~/Documents/beeware/toga/examples/progressbar/progressbar$ python -m app
(venv) yugesh@yugesh-HP-240-G1-Notebook-PC:~/Documents/beeware/toga/examples/progressbar/progressbar$ 

```
But things work when I include that block (is this normal?) - also, all other examples had a separate __main__.py  with `__name__=='__main__'` So i figured it was needed.

Reply:
I think this example isn't working because progressbar is a work in progress for Gtk in PR #291